### DISCUSSION TITLE
2FA Issue when using Security Key YubiKey 5Ci on macOS Sonoma in Safari

### DISCUSSION THREAD

**Discussion by wesleyfranks** [community/community#82184] [Other Feature Feedback, Questions, & Ideas]
Did research into the logs for Safari using the new profiles. I was successfully able to setup a security key in Safari, but it will not successfully let me use it. However, I've tested and the security key works in Edge sometimes. Assuming the same would be for Chrome. I haven't tried Firefox will try it and get back to you. However, what I found seems to be a bug due to the new way apple handles the user sign-in flow. Issue has been noted by YubiCo. 

YubiCo: 
https://support.yubico.com/hc/en-us/articles/360022004600-No-reaction-when-using-WebAuthn-on-macOS-iOS-and-iPadOS

Apple: 
https://webkit.org/blog/11312/meet-face-id-and-touch-id-for-the-web/

I did look for another post similar. I don't believe that post is specifically referring to this issue. Correct me if I'm wrong and I'll just post a reply to: 

Potential Similar Post
https://github.com/orgs/community/discussions/54450

**Comment by PeterFalken** [comment]
I have the same issue and it seems that other have reported it, but no fix has been issued.
